docs/building.md

```diff
@@ -42,6 +42,7 @@ The `merlin` container is sourced by building the container anew using the build
 process. This is evidenced by the `build:` directive in the `merlin`
 section in docker-compose.yml.
 
+Prior to building docker-compose.yml, it is necessary to create a .env file using the template found in .env.template in the Aerie root directory.
```

Comment:
So a `.env` file in a working directory effectively sets environment variables for that working directory? The only mention of Unix `.env` files I can find are ones that reside in `$HOME/.env`, so I believe they fairly global environment variables (per-user).

Reply:
From the [AERIE-1696](https://jira.jpl.nasa.gov/browse/AERIE-1696) ticket I believe we want to [use secrets in compose](https://docs.docker.com/engine/swarm/secrets/#use-secrets-in-compose) instead of environment variables. From the previous link:
>Docker secrets do not set environment variables directly. This was a conscious decision, because environment variables can unintentionally be leaked between containers (for instance, if you use `--link`).

Reply:
I think we should move this PR forward. And look at maybe using secrets later.

Reply:
Okay! In that case I'm still unsure if we're going about this the right way (see 1st comment in this chain).

Reply:
When you run a `docker compose`, Docker looks for a .env in the current working directory and loads it in, as shown by the documentation [here](https://docs.docker.com/compose/environment-variables/):
> If you have multiple environment variables, you can substitute them by adding them to a default environment variable file named .env or by providing a path to your environment variables file using the --env-file command line option.

If the .env was added to the machine's root directory, then they would appear in your global environment, but because it is in a subdirectory, unless explicitly told to nothing in it will be `export`ed. 

While using secrets in compose could also work, we chose to use environment variables instead for compatibility with Amazon's ECS system, [which prefers environment variables](https://aws.amazon.com/premiumsupport/knowledge-center/ecs-data-security-container-task/):
> As a security best practice, pass sensitive information to containers as environment variables.

Reply:
To use Docker secrets it looks like we just need `.txt` files laying around:
```yaml
secrets:
   db_password:
     file: db_password.txt
   db_root_password:
     file: db_root_password.txt
```

Reply:
It's a Docker compose thing: https://docs.docker.com/compose/environment-variables/#the-env-file

Reply:
Ohh gotcha! Thanks for the explanation @Mythicaeda :) That makes a lot of sense.

Reply:
This also works nicely with GH environment secrets

Reply:
No problem @pcrosemurgy!